scrapers/DEU/DE-BY/index.js

```diff
@@ -0,0 +1,77 @@
+import * as fetch from '../../../lib/fetch.js';
+import * as datetime from '../../../lib/datetime.js';
+import * as geography from '../../../lib/geography.js';
+
+// Set county to this if you only have state data, but this isn't the entire state
+// const UNASSIGNED = '(unassigned)';
+
+const scraper = {
+  country: 'DEU',
+  state: 'DE-BY', // ISO 3166 notation
+  url: 'https://raw.githubusercontent.com/jgehrcke/covid-19-germany-gae/master/data.csv',
+  type: 'json',
+  timeseries: true,
+  curators: [
+    {
+      name: 'Dr. Jan-Philip Gehrcke',
+      email: 'jgehrcke@googlemail.com'
+    }
+  ],
+  sources: [
+    {
+      name: 'Robert Koch-Institut, Bavarian health ministry',
+      description: 'Fresh data obtained from Bavarian health ministry by ZEIT ONLINE',
+      url: 'https://github.com/jgehrcke/covid-19-germany-gae'
+    }
+  ],
+  _rowToResult: row => {
+    return {
+      cases: parseInt(row['DE-BY_cases'], 10),
+      deaths: parseInt(row['DE-BY_deaths'], 10),
+      coordinates: [11.497, 48.79],
+      population: 13 * 10 ** 6
+    };
+  },
+  async scraper() {
+    const data = await fetch.csv(this.url);
```

Comment:
If you pass false here, per-day cache is defeated and it uses the local `./cache/` directory. This is required for timeseries since you don't want to refetch the URL or use cached data ever. Sorry I didn't catch this sooner!
```suggestion
    const data = await fetch.csv(this.url, false);
```

Reply:
Thanks! No worries! Pushed a change.

Reply:
(man, I just realize: this GitHub code change suggestion feature looks really cool, it's new to me -- next time I will try to just press the button :).